### PR TITLE
Screensaver: added ITA translation.

### PR DESCRIPTION
**Pull request by SebastianoPistore**
Add italian translation.

### CODE REVIEW

File: modules/rosapps/applications/screensavers/blankscr/scrnsave.rc [hunk 1]
```diff
@@ -59,3 +59,6 @@ IDI_ICON  ICON  DISCARDABLE  "res/icon_blankscr.ico"
 #ifdef LANGUAGE_ZH_CN
     #include "lang/zh-CN.rc"
 #endif
+#ifdef LANGUAGE_IT_IT
```
Comment (HBelusca) [inline modules/rosapps/applications/screensavers/blankscr/scrnsave.rc]: These entries should be placed in alphabetical order (same remark for the other modified .rc files).

File: modules/rosapps/applications/screensavers/cylfrac/cylfrac.rc [hunk 2]
```diff
@@ -30,7 +30,7 @@ IDI_ICON  ICON  DISCARDABLE  "res/icon_cylfrac.ico"
 #ifdef LANGUAGE_ES_ES
     #include "lang/es-ES.rc"
 #endif
-#ifdef LANGUAGE_ES_ES
+#ifdef LANGUAGE_FR_FR
```
Comment (HBelusca) [inline modules/rosapps/applications/screensavers/cylfrac/cylfrac.rc]: Thanks ;)

Comment (SebastianoPistore) [reply modules/rosapps/applications/screensavers/cylfrac/cylfrac.rc]: Even if I'm not a hacker when I read it I realized there was a problem :)

File: modules/rosapps/applications/screensavers/blankscr/scrnsave.rc (hunk 1 above)
Comment (SebastianoPistore) [reply modules/rosapps/applications/screensavers/blankscr/scrnsave.rc]: I did not understand it, I got used to Transifex and I'm no longer used to translating directly from code.

**Comment by SebastianoPistore** [comment]
Now everything should be ok.

**Comment by Fraizeraust** [comment]
Hey there, thank you for your contribution!

While you're still here at it, could you please check #527 (especially the Italian part) if it is alright? Thank you for your reading. :)

File: modules/rosapps/applications/screensavers/blankscr/scrnsave.rc [hunk 3]
```diff
@@ -58,4 +61,4 @@ IDI_ICON  ICON  DISCARDABLE  "res/icon_blankscr.ico"
 #endif
 #ifdef LANGUAGE_ZH_CN
     #include "lang/zh-CN.rc"
-#endif
+#endif
```
Comment (Fraizeraust) [inline modules/rosapps/applications/screensavers/blankscr/scrnsave.rc]: By the way, a blank line after the end of the file is required as per the translation guidelines [here](http://reactos.org/wiki/Translation_Introduction#How_to_translate_the_files).

File: modules/rosapps/applications/screensavers/circles/circles.rc [hunk 4]
```diff
@@ -47,4 +50,4 @@ IDI_ICON  ICON  DISCARDABLE  "res/icon_circles.ico"
 #endif
 #ifdef LANGUAGE_ZH_CN
     #include "lang/zh-CN.rc"
-#endif
+#endif
```
Comment (Fraizeraust) [inline modules/rosapps/applications/screensavers/circles/circles.rc]: Ditto.

File: modules/rosapps/applications/screensavers/cylfrac/cylfrac.rc [hunk 5]
```diff
@@ -59,4 +62,4 @@ IDI_ICON  ICON  DISCARDABLE  "res/icon_cylfrac.ico"
 #endif
 #ifdef LANGUAGE_ZH_CN
     #include "lang/zh-CN.rc"
-#endif
+#endif
```
Comment (Fraizeraust) [inline modules/rosapps/applications/screensavers/cylfrac/cylfrac.rc]: Ditto.

File: modules/rosapps/applications/screensavers/starfield/starfield.rc [hunk 6]
```diff
@@ -56,4 +59,4 @@ IDI_ICON  ICON  DISCARDABLE  "res/icon_starfield.ico"
 #endif
 #ifdef LANGUAGE_ZH_CN
     #include "lang/zh-CN.rc"
-#endif
+#endif
```
Comment (Fraizeraust) [inline modules/rosapps/applications/screensavers/starfield/starfield.rc]: Ditto.

**Pull request merged**